### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Vasilije1990**
Added PromethAI, personal assistant focused on nutrition

For more info, check [here](www.prometh.ai)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -696,7 +696,7 @@ An open-source autonomous AI framework to enable development and deployment auto
 
 ### Description
 - An AI agent framework
-- Open source, but infrastructure is closed-source
+- Open source, but infrastructure is -source
```
Comment (tizkovatereza) [inline README.md:699]: Could you please provide more explanation on the changes in this line?

Comment (Vasilije1990) [reply README.md]: Should be reverted. Sorry, an issue on my side

**Review by tizkovatereza — APPROVED** [review]
Thank you for contributing! Would you mind adding even more info about your agent, e.g. webpage, and a summary of how it works?

**Comment by Vasilije1990** [comment]
> Thank you for contributing! Would you mind adding even more info about your agent, e.g. webpage, and a summary of how it works?

Will expand today!

**Pull request merged**